storage/database/db_manager.go

```diff
@@ -1797,6 +1816,44 @@ func (dbm *databaseManager) WriteChainConfig(hash common.Hash, cfg *params.Chain
 	}
 }
 
+// ReadAccountSnapshot retrieves the snapshot entry of an account trie leaf.
+func (dbm *databaseManager) ReadAccountSnapshot(hash common.Hash) []byte {
+	db := dbm.getDatabase(SnapshotDB)
+	data, _ := db.Get(AccountSnapshotKey(hash))
```

Comment:
Why don't we implement readAccountSnapshot()?

Reply:
The `writeAccountSnapshot` is for batch write as well as a single(?) write. Do you think `readAccountSnapshot` is needed?

Reply:
I'm not sure it is necessary, but it seems little bit strange to me because only this function seems not to have a wrapper. :)

Reply:
@kjhman21 Adding a wrapper is okay, but still a bit ugly if we do not intriduce `KeyValueReader`. Well, the wrapper function is with `KeyValueWriter` struct and we don't have `KeyValueReader` yet. Introducing this structure is a bit burdensome on this PR. Let me do it later then.